examples/contrib/readhar.py

```diff
@@ -0,0 +1,134 @@
+"""Reads HAR files into flow objects"""
+import asyncio
+import codecs
+import json
+import logging
+import time
+
+from mitmproxy import command
+from mitmproxy import connection
+from mitmproxy import ctx
+from mitmproxy import exceptions
+from mitmproxy import http
+from mitmproxy import types
+
+
+class ReadHar:
+    def __init__(self):
+        self.harFile = {}
+        self.flows = []
+
+    def fix_headers(self, request_headers):
+        """Changes provided headers into flow-friendly format"""
+        flow_headers = []
+        for header in request_headers:
+            if type(header) == dict and header.get("name", None):
+                key = header["name"]
+                value = header["value"]
+            else:
```

Comment:
Based on the code I assume that there are HAR files with `{name: ..., value: ...}` dicts as well as HAR files with `[name, value]` arrays? For future readers it's useful to add a rough comment explaining the context. For example, if all HAR files but Firefox ones use the former notation, a `# Firefox's HAR files deviate by doing X...`-style comment would be useful.